### PR TITLE
update options table in readme

### PR DESCRIPTION
**Pull request by UziTech**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [x] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

update defaults on readme to v4 defaults

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,10 +113,11 @@ module.exports = {
 | :-----------------------------------: | :-------------------------: | :----------------: | :--------------------------------------------------------------------- |
 |           **[`url`](#url)**           |    `{Boolean\|Function}`    |       `true`       | Enables/Disables `url`/`image-set` functions handling                  |
 |        **[`import`](#import)**        |    `{Boolean\|Function}`    |       `true`       | Enables/Disables `@import` at-rules handling                           |
-|       **[`modules`](#modules)**       | `{Boolean\|String\|Object}` |      `false`       | Enables/Disables CSS Modules and their configuration                   |
+|       **[`modules`](#modules)**       | `{Boolean\|String\|Object}` |   `{auto: true}`   | Enables/Disables CSS Modules and their configuration                   |
+|          **[`icss`](#icss)**          |         `{Boolean}`         |    `!!modules`     | Enables/Disables Interoperable CSS                                     |
```
Comment (UziTech) [inline README.md:117]: I don't know the best best way to indicate that the default value for `icss` depends on the `modules` option.

Comment (evilebottnawi) [reply README.md:117]: Rewrite it `false by default, otherwise depend on `modules``

**Comment by UziTech** [comment]
failing tests are unrelated to this PR

**Comment by evilebottnawi** [comment]
Thanks!

**Pull request merged**